package.json

```diff
@@ -8,7 +8,7 @@
     "start": "npm run build && node dist/index.js",
     "test": "jest --silent --coverage",
     "lint": "npx prettier --check .",
-    "lint-fix": "npx prettier --write .",
```

Comment:
is there a  'terraform fmt check' type command that can be added here as an '&& <your command here>'

Reply:
will investigate